src/main/scala/viper/gobra/reporting/DefaultErrorBackTranslator.scala

```diff
@@ -64,22 +64,23 @@ object DefaultErrorBackTranslator {
       case vprrea.NegativePermission(CertainSource(info)) =>
         NegativePermissionReason(info)
       //      case vprrea.InvalidPermMultiplication(offendingNode) =>
-      //      case vprrea.MagicWandChunkNotFound(offendingNode) =>
-      //      case vprrea.NamedMagicWandChunkNotFound(offendingNode) =>
-      //      case vprrea.MagicWandChunkOutdated(offendingNode) =>
+      case vprrea.MagicWandChunkNotFound(CertainSource(info)) =>
+        MagicWandChunkNotFound(info)
       case vprrea.ReceiverNotInjective(CertainSource(info)) =>
         ReceiverNotInjectiveReason(info)
       //      case vprrea.LabelledStateNotReached(offendingNode) =>
```

Comment:
this comment is no longer needed. You could however move the last case here such that we keep the same ordering as in silver